nfd/README.md

```diff
@@ -1,22 +1,27 @@
 ## NFD
 
-The [Node Feature Discovery](https://github.com/openshift/node-feature-discovery) (NFD) manages the detection of hardware features and configuration in OCP cluster by labeling the nodes with hardware-specific information.
+The [Node Feature Discovery](https://github.com/openshift/node-feature-discovery) (NFD) manages the detection of hardware features and configuration in a OCP cluster by labeling the nodes with hardware-specific information.
 
-NFD labels the host with node-specific attributes, such as PCI cards, kernel, operating system version, and so on. In this project NFD is used to automatically label nodes with specific Hardware features like Intel dGPU card, Intel SGX and Intel QAT etc.
+NFD labels the host with node-specific attributes, such as PCI cards, kernel, operating system version, and so on. In this project, NFD is used to automatically label nodes with specific Intel® hardware features such as Intel® Data Center GPU Flex Series, Intel® SGX and Intel® QAT etc.
```

Comment:
Can we change Intel(r) hardware features to hardware features? I think NFD can also be used to label other hardware features, Right? :=)

Reply:
Done